src/sentry/conf/server.py

```diff
@@ -1780,15 +1783,22 @@ def build_cdc_postgres_init_db_volume(settings):
     ),
     "clickhouse": lambda settings, options: (
         {
-            "image": "yandex/clickhouse-server:20.3.9.70",
+            # altinity provides clickhouse support to other companies
+            # Official support: https://github.com/ClickHouse/ClickHouse/issues/22222
+            # This image is build with this script https://gist.github.com/filimonov/5f9732909ff66d5d0a65b8283382590d
+            "image": "altinity/clickhouse-server:21.6.1.6734-testing-arm"
```

Comment:
Is this the earliest ClickHouse version that works with ARM? That's a big jump from 20.3 to 21.6.